test/rollup.config.js

```diff
@@ -8,7 +8,32 @@ export default {
     format: 'cjs',
   },
   plugins: [
-    serve({ contentBase: '', port: Math.round(Math.random() * 10000) + 40000 }),
-    live(),
+    serve({
```

Comment:
Do you have an idea of how to test this in CI?

Reply:
hi @thgh, do you mean testing plugin serve with that specific configuration?

I supose we could set a GH action with node.js that captures the output of rollup... haven't done it before though

Reply:
Had a peek at how node-livereload does the server testing. 

https://github.com/frantic0/node-livereload/blob/main/test/index.test.coffee

It is a travis CI setup and does include server configurations

Reply:
No requirement, just looking for inspiration

I'm not using this plugin actively so we should start doing automated testing to remain confident when releasing. I hope to make it a requirement soon.